### ISSUE TITLE
Null reference exception

### ISSUE THREAD

**Issue by conficient** [mrlacey/ConstVisualizer#9]
Seeing this in the console
```
Error in ReloadConstsAsync
Object reference not set to an instance of an object.
ConstVisualizer
   at ConstVisualizer.ConstFinder.<TrackConstsInDocumentAsync>d__11.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at ConstVisualizer.ConstFinder.<ReloadConstsAsync>d__10.MoveNext()
```

**Comment by mrlacey** [comment]
@conficient  Is this in VS2019 or 2022?

**Comment by conficient** [comment]
VS2022

**Comment by mrlacey** [comment]
I've just released v1.13 to the marketplace which will hopefully fix this.

I've noticed that VS2022 has changed the way it handles some file types and project property pages when opened.
If you're able to tell what you were doing (what type of document or windows you had opened when this happened, I may be able to do something more specific to try and prevent such issues in future.
